index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;300;400;500;600;700&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header">
+        <nav class="nav">
+            <img class="pic" src="Union.png" alt="HamburgerMenu">
+            <h4 class="name">Shuja</h4>
+        </nav>
```

Comment:
Nice work so far @shuja-shah.✋

- [ ] [Optional] You may the union, the logo, and the hamburger icon inside `<a>` tags. This is because they are clickable links